arrow-ipc/src/writer.rs

```diff
@@ -967,12 +1074,15 @@ fn write_continuation<W: Write>(
 }
 
 /// In V4, null types have no validity bitmap
-/// In V5 and later, null and union types have no validity bitmap
+/// In V5 and later, null, union and run end encoded types have no validity bitmap
```

Comment:
Although the result is correct, the doc reads like run end encoded type has validity bitmap before bitmap. 😅 

```suggestion
/// In V5 and later, null and union types have no validity bitmap
/// Run end encoded types has no validity bitmap.
```